doc/ug/getting_started_verilator.md

```diff
@@ -165,8 +165,8 @@ Tracing slows down the simulation by roughly factor of 1000.
 ```console
 $ cd $REPO_TOP
 $ build/lowrisc_systems_top_earlgrey_verilator_0.1/sim-verilator/Vtop_earlgrey_verilator \
-  --meminit=rom,sw/device/sim_boot_rom/rom.elf \
-  --meminit=flash,sw/device/sim_hello_world/sw.elf \
+  --meminit=rom,build-bin/sw/device/sim-verilator/boot_rom/boot_rom.elf \
+  --meminit=flash,build-bin/sw/device/sim-verilator/examples/hello_world/hello_world.elf
```

Comment:
You should use build-out paths here, the build-bin variants don't have a stable directory structure.

Reply:
Thanks for pointing that out Philipp!

Reply:
I think you got that backwards, build-out is where Meson dumps stuff, build-bin is where exports happen.

Reply:
Ah, okay it is indeed `build-bin` where we should get the binaries from. This is documented here: https://docs.opentitan.org/doc/ug/getting_started_sw/

Reply:
Turns out the naming is indeed confusing.